app/models/session.rb

```diff
@@ -288,6 +288,15 @@ def open_for_consent?
     close_consent_at&.today? || close_consent_at&.future? || false
   end
 
+  def next_reminder_date = next_reminder_dates&.first
```

Comment:
```suggestion
  def next_reminder_date = next_reminder_dates.first
```

Can we make `next_reminder_dates` return `[]` if there aren't any? Then we don't need to handle two return types.